### ISSUE TITLE
[Feature Request] Batch Renormalization

### ISSUE THREAD

**Issue by arthurflor23** [tensorflow/tensorflow#69790]
### Issue type

Feature Request

### Have you reproduced the bug with TensorFlow Nightly?

Yes

### Source

source

### TensorFlow version

2.16.1

### Custom code

No

### Current behavior?

Hi,

The `renorm` parameter in the `BatchNormalization` layer, which was available in TensorFlow old versions (such as 2.5.0: [link](https://github.com/tensorflow/tensorflow/blob/a4dfb8d1a71385bd6d122e4f27f86dcebb96712d/tensorflow/python/keras/layers/normalization.py#L99)), doesn't exist in the current versions (such as 2.16.1).

I don't if it was removed for a specific reason, but could you please consider reintroducing the`renorm` option in the`BatchNormalization` layer ?

In any case, thank you in advance.

### Standalone code to reproduce the issue

```shell
Tensorflow 2.5.0:
tf.keras.layers.BatchNormalization(renorm=True)(...)

Tensorflow 2.16.1:
tf.keras.layers.BatchNormalization()(...)
```

**Comment by tilakrayal** [comment]
@arthurflor23,
**Renorm** is not implemented in any of the backend and it is not in the list of allowed arguments of BatchNormalization here https://keras.io/api/layers/normalization_layers/batch_normalization/

In keras 3, we removed some the flags from BN layer that was previously backend specific (eg renorm, virtual batch size, etc). So the renorm param is not supported by the tf backend as well.

Also please take a look at this comment for the similar feature. 
https://github.com/keras-team/keras/issues/19063#issuecomment-1917629801
Thank you!
